src/plugins/kukemc-beautify/index.tsx

```diff
@@ -145,10 +142,57 @@ function groundGlass(): void {
     }
     styleElement.innerText = `.gandi-stage-wrapper{backdrop-filter: none;}.gandi_stage-header_stage-header-wrapper_1F4gT{backdrop-filter: blur(${ambiguity}px);}.gandi_collapsible-box_header_dc9Es{border-top-left-radius: 7px;border-top-right-radius: 7px;}`;
   } catch (error) {
-    console.error("Error applying ground glass effect:", error);
+    console.error("Error applying frosted glass effect:", error);
   }
 }
 
+function removeFrostedGlass(): void {
+  const styleElements = document.head.getElementsByTagName("style");
+
+  const htmlElement = document.documentElement;
+  const computedStyle = window.getComputedStyle(htmlElement);
+  const themeColorHex = computedStyle.getPropertyValue("--theme-color-300").trim(); // Remove any potential whitespace
+
+  const themeColorRgb = hexToRgb(themeColorHex);
+
+  const { r, g, b } = themeColorRgb;
+
+  try {
+    for (let i = 0; i < styleElements.length; i++) {
+      const sheet = styleElements[i].sheet as CSSStyleSheet;
+      if (sheet.cssRules) {
+        for (let j = 0; j < sheet.cssRules.length; j++) {
+          const rule = sheet.cssRules[j] as CSSStyleRule;
+          const isTargetSelector = [
+            ".blocklyWidgetDiv .goog-menu",
+            ".gandi_context-menu_context-menu_2SJM-",
+            ".blocklyToolboxDiv",
+            ".gandi_plugins_plugins-root_xA3t3",
+            ".gandi_collapsible-box_collapsible-box_1_329",
+            ".gandi_target-pane_count_3fmUd",
+            ".gandi_editor-wrapper_tabList_4HFZz",
+            ".gandi_setting-modal_modal-overlay_3wJji",
+            ".gandi_bulletin-modal_modal-overlay_TBAhj",
+            ".gandi_collapsible-box_header_dc9Es",
+          ].includes(rule.selectorText);
+
+          if (rule.selectorText === ".gandi_setting-modal_modal-overlay_3wJji") {
+            applyStyleChanges(rule, `rgba(0,0,0,.7)`, `blur(0)`);
+          } else if (rule.selectorText === ".blocklyToolboxDiv") {
+            applyStyleChanges(rule, `rgba(${r}, ${g}, ${b}, 1)`, `blur(0)`);
+          } else if (isTargetSelector) {
+            applyStyleChanges(rule, `rgba(${r}, ${g}, ${b}, 1)`, `blur(0)`);
+          }
+        }
+      }
+    }
+    styleElement.innerText = `.gandi_collapsible-box_header_dc9Es{border-top-left-radius: 7px;border-top-right-radius: 7px;}`;
```

Comment:
see **frostedGlass** function
there`s an EventListener has been added
i think u have to remove it here